CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [5.0.3]
+### Changed
+- Standardised repo per module template as of April 2023 ([#102](https://github.com/MetaMask/utils/pull/102))
```

Comment:
It looks like this PR introduced a couple of breaking changes: the minimum Node version got bumped to 16, and the JavaScript shipped in the package was changed to be ES2020-compatible rather than ES2017-compatible.

Reply:
Yeah, I agree.